### PR TITLE
ocrvs-3295: record audit's history's actionDetailsModal bug fixed for comments

### PR DESCRIPTION
**Pull request by emon-swe-sust**
Screenshot from 2022-06-02 12-09-51

### CODE REVIEW

File: packages/client/src/views/RecordAudit/ActionDetailsModal.tsx [hunk 1]
```diff
@@ -241,7 +241,7 @@ export const ActionDetailsModalListTable = ({
         columns={commentsColumn}
         content={actionDetailsData.comments
           .concat(actionDetailsData.statusReason?.text || [])
-          .map((text: string) => ({ comment: text }))}
+          .map((text: any) => ({ comment: text.comment }))}
```
Comment (rikukissa) [inline packages/client/src/views/RecordAudit/ActionDetailsModal.tsx:244]: I'm guessing this bug was caused by incorrect typing. Wouldn't it make sense to fix the typing to ensure it doesn't happen again when someone changes the code?

Comment (rikukissa) [inline packages/client/src/views/RecordAudit/ActionDetailsModal.tsx:244]: The absolute minimum would probably be 

```suggestion
          .map((text: { comment: string }) => ({ comment: text.comment }))}
```

`any` indicates that the value can be literally anything, including number, boolean... that do not have a `comment` property

Comment (emon-swe-sust) [reply packages/client/src/views/RecordAudit/ActionDetailsModal.tsx:244]: Yes. Absolutely make sense.

**Review by rikukissa — APPROVED** [review]

**Comment by euanmillar** [comment]
Probably a conflict with @yeasinhossain-dsi PR when he fixed death comments

**Comment by emon-swe-sust** [comment]
> Probably a conflict with @yeasinhossain-dsi PR when he fixed death comments

And, the bug for this PR is also fixed by @yeasinhossain-dsi's PR. And I think this PR is no longer valid.

**Pull request closed without merging**